snowplow_tracker/events/pageview.py

```diff
@@ -0,0 +1,76 @@
+# """
+#     pageview.py
+
+#     Copyright (c) 2013-2023 Snowplow Analytics Ltd. All rights reserved.
+
+#     This program is licensed to you under the Apache License Version 2.0,
+#     and you may not use this file except in compliance with the Apache License
+#     Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+#     http://www.apache.org/licenses/LICENSE-2.0.
+
+#     Unless required by applicable law or agreed to in writing,
+#     software distributed under the Apache License Version 2.0 is distributed on
+#     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#     express or implied. See the Apache License Version 2.0 for the specific
+#     language governing permissions and limitations there under.
+# """
+from snowplow_tracker.events.event import Event
+from typing import Optional
+
+
+class Pageview(Event):
+    def __init__(
+        self,
+        page_url: Optional[str] = None,
+        page_title: Optional[str] = None,
+        referrer: Optional[str] = None,
+    ) -> None:
+        """
+        :param  page_url:       URL of the viewed page
+        :type   page_url:       non_empty_string
+        :param  page_title:     Title of the viewed page
+        :type   page_title:     string_or_none
+        :param  referrer:       Referrer of the page
+        :type   referrer:       string_or_none
+        """
+        super(Pageview, self).__init__()
+        self.pb.add("e", "pv")
+        self.page_url = page_url
+        self.page_title = page_title
+        self.referrer = referrer
+
+    @property
+    def page_url(self) -> Optional[str]:
+        """
+        URL of the viewed page
+        """
+        return self._page_url
+
+    @page_url.setter
+    def page_url(self, value: Optional[str]):
+        self._page_url = value
+        self.pb.add("url", self._page_url)
+
+    @property
+    def page_title(self) -> Optional[str]:
+        """
+        Title of the viewed page
+        """
+        return self._page_title
```

Comment:
I think we can simplify this a bit and not use the extra `self._page_title` property but directly read from the payload:

```suggestion
        return self.pb.get("page")
```